### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Mattee9**
Adding more classes

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/Hard.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package edu.oregonstate.cs361.battleship;
+
+/**
+ * hmm
+ */
+
+public class Hard extends BattleshipModel{
+    public Hard(){}
+}
```
Comment (ksteinfeldt) [inline src/main/java/edu/oregonstate/cs361/battleship/Hard.java:9]: This seems to be a problem?

File: src/main/java/edu/oregonstate/cs361/battleship/Player.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package edu.oregonstate.cs361.battleship;
+
+/**
+ * Created by Dennie on 3/15/2017.
+ */
+
+public class Player {
+
+
+    public coord Firsthit;
```
Comment (johnbren85) [inline src/main/java/edu/oregonstate/cs361/battleship/Player.java:10]: Looks like exactly what we asked for, thanks.

**Comment by johnbren85** [comment]
Looks good, thanks

**Review by ksteinfeldt — COMMENTED** [review]
Looks good now. We'll have to fill it out as we go, but good start.

**Pull request merged**